roles/gunicorn/templates/djangoblog.conf.j2

```diff
@@ -4,9 +4,12 @@ start on (filesystem)
 stop on runlevel [016]
 
 respawn
-setuid nobody
-setgid nogroup
+setuid ubuntu
+setgid ubuntu
 chdir {{ django_home }}
-env DJANGO_SETTINGS_MODULE=mysite.production
+env IMAGER_SECRET_KEY='i4jz)*^p(o@s32p1)wdq@yonk$d+pdx_b0t%b4(7vn4hztdwrf'
```

Comment:
this is still in github, you know.  Could you template this in like you do with `{{django_home}}`?